src/Root.cpp

```diff
@@ -423,7 +423,20 @@ void cRoot::LoadWorlds(cSettingsRepositoryInterface & a_Settings, bool a_IsNewIn
 			continue;
 		}
 		FoundAdditionalWorlds = true;
-		cWorld * NewWorld = new cWorld(WorldName.c_str());
+		cWorld * NewWorld;
+		// Note that if there's a world.ini file present, the world will override the following worldTypes in cWorld::start
+		if (WorldName.find("_nether") != std::string::npos)  // If the world has the word nether, mark it as a nether world. (will be overridden by setting  if ini present)
```

Comment:
AString is an xoft-ism (i.e. completely unnecessary type definition) for the standard string type.


On 3 Feb 2016, at 22:15, "Julian Laubstein" <notifications@github.com> wrote:

> In src/Root.cpp:
> 
> > @@ -423,7 +423,20 @@ void cRoot::LoadWorlds(cSettingsRepositoryInterface & a_Settings, bool a_IsNewIn
> >  			continue;
> >  		}
> >  		FoundAdditionalWorlds = true;
> > -		cWorld * NewWorld = new cWorld(WorldName.c_str());
> > +		cWorld * NewWorld;
> > +		// Note that if there's a world.ini file present, the world will override the following worldTypes in cWorld::start
> > +		if (WorldName.find("_nether") != std::string::npos)  // If the world has the word nether, mark it as a nether world. (will be overridden by setting  if ini present)
> I think we could implement a method like this:
> 
> bool AString::contains(char*);
> —
> Reply to this email directly or view it on GitHub.
>